README.md

```diff
@@ -244,6 +244,30 @@ Returns `true` if the message could be decrypted. Otherwise `false`.
 
 The decrypted message will be stored in `message`.
 
+### Sealed box encryption
+
+Bindings for the crypto_box_seal API.
+[See the libsodium crypto_box_seal docs for more information](https://download.libsodium.org/doc/public-key_cryptography/sealed_boxes.html).
+
+#### `crypto_box_seal(cipher, message, publicKey)`
+
+Encrypt a message in a sealed box using a throwaway keypair.
+The ciphertext cannot be associated with the sender due to the sender's key
+being a single use keypair that is overwritten during encryption.
+
+* `cipher` should be a buffer with length at least `message.length + crypto_box_SEALBYTES`.
+* `message` should be a buffer with any length.
+* `publicKey` should be the receipent's public key.
```

Comment:
I'd like to note that the keys have to be of the `crypto_box_*` kind since there are so many different keypairs in sodium :) I know it right there in the sodium docs, but it would be nice to have it here too